### DISCUSSION TITLE
Filtering accounts to be displayed.

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#3577] [Q&A] [answered]
Hello everyone, 

I've been researching how to filter accounts and have successfully shown only active accounts. However, I need to further refine the filter to exclude conference room accounts. These accounts are in Azure and not in my on-site AD.

Using the **SearchQueryToolv**, I validated the user properties and observed that these users do NOT have a Department assigned. However, I'm unsure how to filter to only display users with a Department assigned, meaning the property is blank or empty.

Any ideas?

Thanks for your help.

**Comment by kasperbolarsen** [comment, marked as answer]
The guidelines here , https://microsoft-search.github.io/pnp-modern-search/scenarios/Create-a-useful-People-Search/ does show how to filter on Department  :-)

**Comment by wobba** [comment]
`Department:*` should show all with a value. `NOT Department:*` should show without a value,.

**Comment by adumith** [comment]
I must confess that I have tried both options and the one that worked for me was the one in the documentation.

I mean this one:
**{inputQueryText} OfficeNumber:{verticals.value} "SPS-HideFromAddressLists"=0 owstaxIdSPShDepartment:"#8ed8c9ea-7052-4c1d-a4d7-b9c10bffea6f"**


Thank you very much for your help.
